### DISCUSSION TITLE
"Inverted" Colors on Fresh Install

### DISCUSSION THREAD

**Discussion by MylesDev** [shadcn-ui/ui#4791] [Q&A]
Hello,

I have a fresh NextJS app & shadcn project (using `npx shadcn@latest init sidebar-01 login-01 -d -f -y`), the install seems to have no issues.

However, when going to `http://localhost:3000/dashboard` after running `npm run dev` - the colors are completely off, and installing other blocks (such as chart-01) result in the colors becoming "inverted".

Additionally, at least for `/dashboard` I can temporarily see the correct page before it becomes like the screenshot below. After which the console message "Layout was forced before the page was fully loaded. If stylesheets are not yet loaded this may cause a flash of unstyled content."

Here's what I see when going to the `/dashboard` endpoint.
![71ce5db28201239f6f8b603b](https://github.com/user-attachments/assets/6b00fcf1-52b4-4547-b9d1-1e159d8c8d2c)

Unsure if I have done something wrong, as I have just followed the guide/shadcn's CLI tweet, but absolutely no idea on how to tackle this issue from here as I'm fairly new to front end development in general.

Any help would be appreciated!

**Comment by MylesDev** [comment]
Managed to resolve this by removing the following from my `global.css` file:
`@media (prefers-color-scheme: dark) {
    :root {
        --background: #0a0a0a;
        --foreground: #ededed;
    }
}`

Whilst that has resolved my issue, I'd be interested to know why?

**Comment by QuentinFrc** [comment]
I made an issue #4801 and a related PR to fix that #4802.

It seems that the "inverted" rendering is due to a bg-accent/50 on the sidebar layout ([Here in the block](https://github.com/shadcn-ui/ui/blob/f4ca57a79cf2d56f9c55021242a55cf0e1018b72/apps/www/registry/default/block/sidebar-01/ui/sidebar.tsx#L57)).
The PR simply removes the background on the SidebarLayout (for both registries)
